tests/staticfiles_tests/test_management.py

```diff
@@ -130,6 +132,18 @@ def test_local_storage_detection_helper(self):
             storage.staticfiles_storage = staticfiles_storage
 
 
+class TestCollectionsHelpSubcommand(AdminScriptTestCase):
+    @override_settings(STATIC_ROOT=None)
+    def test_missing_settings_dont_prevent_help(self):
+        """
+        Even if the :setting:`STATIC_ROOT` is not set, one can still call the
```

Comment:
The committer can amend this when merging, but there's no advantage in using restructedtext markup like ":setting:" in docstrings.